.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost
```

Comment:
this file shouldnt be in the repo. Add it to the gitignore and delete it from here